Extensions/AdMob/admobtools.ts

```diff
@@ -108,22 +109,53 @@ namespace gdjs {
     let rewardedVideoRewardReceived = false; // Becomes true when the video is closed and the reward is received.
     let rewardedVideoErrored = false; // Becomes true when the video fails to load.
 
-    let npaValue = '0'; // TODO: expose an API to change this and also an automatic way using the consent SDK.
+    let npaValue = '0'; // 0 means that the user has consented to personalized ads, 1 means that the user has not consented to personalized ads.
 
     // Admob initialization listener
     document.addEventListener(
       'deviceready',
       async () => {
-        // Obtain user consent ?
-
         logger.info('Starting AdMob.');
         isStarting = true;
 
-        await admob.start();
+        if (cordova.platformId === 'ios') {
```

Comment:
I wonder if it's a good idea to delay this by 2 seconds, and offer:
- an action to cancel automatic consent dialog/tracking authorization display.
- an action to do it manually.

So that if I want to postpone this (because it's better if my player plays a bit or click a button in the menu, so I have the opportunity to tell them "we'll ask for your consent, please say yes!"), I can do by:
-  using the first action at launch of my game
- using the second action whenever I want.

Reply:
Possibly yes. 
I've seen issues about users' apps declined by Apple because they couldn't see the consent, and something about it being show too quick so not visible or conflicting with others: https://github.com/admob-plus/admob-plus/issues/652